### PR TITLE
Travis and tests

### PR DESCRIPTION
**Pull request by MarioBlazek**
This pull request adds Travis CI and CodeCov configurations. Also two Composer scripts are added to composer.json file:
* test -> for running PHPUnit tests
* fix -> runs PHP CS Fixer with new configuration (configuration is also include in PR)

Please enable Travis CI and CodeCov services for Github repository before merging this PR.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,10 +26,17 @@
     "require-dev": {
         "matthiasnoback/symfony-config-test": "~2.0",
         "matthiasnoback/symfony-dependency-injection-test": "~1.0",
-        "phpunit/phpunit": "^5.7"
+        "phpunit/phpunit": "^5.7",
+        "friendsofphp/php-cs-fixer": "~2.0"
```
Comment (emodric) [inline composer.json]: Is this really needed?

Comment (MarioBlazek) [inline composer.json]: It is required for composer fix script. Nice to have feature :)

Comment (emodric) [inline composer.json]: :+1:

**Comment by emodric** [comment]
Enabled Travis and CodeCov, trigger the build somehow :)

**Pull request merged**